src/main/steps/edge-case/upload-other-information/content.ts

```diff
@@ -5,15 +5,10 @@ import { ResourceReader } from '../../../modules/resourcereader/ResourceReader';
 export const form: FormContent = {
   fields: () => {
     return {
-      documentRelevance: {
-        type: 'text',
-        label: l => l.title3,
-        classes: 'govuk-text',
-        hint: h => h.line4,
-      },
       additionalInformation: {
         type: 'textarea',
         classes: 'govuk-textarea',
+        autocomplete: 'on',
```

Comment:
Why was this change made?

Reply:
A failed attempt to make it so that the input into the additionalInformation would not be erased on file upload; another solution has been coded into the template.njk for the additional documents page now